site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/SupportForm.js

```diff
@@ -0,0 +1,297 @@
+import { PropTypes } from "prop-types";
+import React, { Component } from "react";
+import { Formik, Form } from "formik";
+import { Form as SemanticForm, Button, Modal } from "semantic-ui-react";
+import { TextField, TextAreaField, ToggleField } from "react-invenio-forms";
+import CategoryDropdown from "./CategoryDropdown";
+import axios from "axios";
+import Dropzone from "react-dropzone";
+import FileTable from "./FileTable";
+import { remove, set } from "lodash";
+import { humanReadableBytes } from "react-invenio-deposit";
+
+// TODO translations
+// TODO implement error notification
+// TODO implement recaptcha
+// TODO required fields are highlighted by the component and not Formik
+
+const apiConfig = {
+    withCredentials: true,
+    xsrfCookieName: "csrftoken",
+    xsrfHeaderName: "X-CSRFToken",
+    headers: {
+        "Accept": "application/json",
+        'Content-Type': 'multipart/form-data',
+    },
+};
+
+const http = axios.create(apiConfig);
```

Comment:
major: We centralized [axios](https://github.com/inveniosoftware/react-invenio-forms/blob/master/src/lib/api/connector.js#L25) so you that one...You need only to modify the headers for your content type only!